MAUI/Themes/Keys.md

```diff
@@ -294,4 +294,1497 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+        </td>
+    </tr>  
+	<tr>
+    <td>
+            SfCalendarStartRangeSelectionColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of start range selection.
+            <br/>
+            <br/>
+        </td>
+    </tr>  
+	<tr>
+    <td>
+            SfCalendarEndRangeSelectionColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of end range selection.
+            <br/>
+            <br/>
+        </td>
+    </tr>  
+	<tr>
+    <td>
+            SfCalendarRangeSelectionColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of range selection.
+            <br/>
+            <br/>
+        </td>
+    </tr>  
+	<tr>
+    <td>
+            SfCalendarHeaderHoverTextColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of text when hovering over the header
+            <br/>
+            <br/>
+        </td>
+    </tr>  
+	<tr>
+    <td>
+            SfCalendarButtonTextColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of button text.
+            <br/>
+            <br/>
+        </td>
+    </tr>  
+	<tr>
+    <td>
+            SfCalendarDisabledNavigationArrowColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of disabled navigation arrow.
+            <br/>
+            <br/>
+        </td>
+    </tr>  
+	<tr>
+    <td>
+            SfCalendarHoverColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of hover.
+            <br/>
+            <br/>
+        </td>
+    </tr>  
+    <tr>
+    <td>
+            SfCalendarSpecialDayIconColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of special day icon.
+            <br/>
+            <br/>
+        </td>
+    </tr>  	
+    <tr>
+    <td>
+            SfCalendarNormalBackground          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of calendar normal background.
+            <br/>
+            <br/>
+        </td>
+    </tr>  	
+</table>
+
+## SfBackdrop
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="14">
+            SfBackdropStyles  
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfBackdropPageTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfBackdrop without merging common theme resource and control style resource dictionaries.
```

Comment:
By merging this key in application resources, you can customize the appearance of SfBackdrop without merging common theme resources and control style resource dictionaries.